packages/web/src/scss/components/Grid/_tools.scss

```diff
@@ -1,20 +1,45 @@
 @use 'sass:map';
+@use 'sass:string';
 @use '@tokens' as tokens;
 @use '../../tools/breakpoint';
 
-// Generates grid-gap values for each breakpoint
+// Generates column-gap and row-gap values for each breakpoint
 // Parameters are:
-// * $gutters: the gutters map
+// * $gutters: the gaps map
 // * $breakpoints: the breakpoints map
 @mixin gutters($gutters, $breakpoints) {
+    $gap-fallback: map.get($gutters, 'mobile');
+
+    $column-property-name: '--grid-spacing-x';
+    $row-property-name: '--grid-spacing-y';
+
+    $column-fallback: var(--grid-default-gap, #{$gap-fallback});
+    $row-fallback: var(--grid-default-gap, #{$gap-fallback});
+
     @each $breakpoint-name, $breakpoint-value in $breakpoints {
         @if not map.has-key($gutters, $breakpoint-name) {
             @error 'Invalid breakpoint specified! #{$breakpoint-name} doesn\'t exist. Use one of #{map.keys($breakpoints)}';
         }
+
+        @if $breakpoint-value > 0 {
+            // stylelint-disable custom-property-no-missing-var-function -- We are only building the fallback value here.
```

Comment:
In my opinion it is no longer needed (?)

Reply:
You're right, thanks. :)

Reply:
That's a little bit confusing, because if I omit this comment Stylelint is ✅.
But if I would move this code directly to the `_Grid.scss` (like it is in your case) it doesn't pass this linting rule… 🤔